app/views/ErrorPage.scala.html

```diff
@@ -52,10 +53,11 @@ <h1 id="title" class="govuk-heading-xl">
      @isCurrentTaxYear match {
             case _ if(code == 63091 || code == 65127 || code == 63085) => {
                 @Html(messages("ServiceMessage." + (code)))
-            }
+                <script @{CSPNonce.attr}>document.querySelectorAll("#back-link-63091, #link-try-again").forEach(elem => elem.addEventListener("click", () => window.history.go(-1)));</script>
```

Comment:
was there a #back-link-63091 on the page before? Is there one now? I can't see it. Also, there's a script tag for #link-try-again below so why do we need it here too?

Reply:
oh, i see #back-link-63091 href is in the messages file.

Reply:
yes some messages have html tags in them and is one of them. There're a few I don't want to change and leave them as is, at the same time apply CSP. so given the html tag an id attribute and if that error occurs, dynamically a script is added so the page doesn't violate the CSP